x-pack/legacy/plugins/ml/public/data_frame/pages/transform_management/components/transform_list/actions.tsx

```diff
@@ -26,7 +26,7 @@ export const getActions = () => {
     {
       isPrimary: true,
       render: (item: DataFrameTransformListRow) => {
-        if (item.stats.task_state !== DATA_FRAME_TASK_STATE.STARTED) {
+        if (item.stats.state === DATA_FRAME_TRANSFORM_STATE.STOPPED) {
```

Comment:
Why did this change from checking `STARTED` to `STOPPED`?

Reply:
The reason is that state now can return more different strings like `indexing`, `stopping`, `aborting` which all mean it's more or less still running. So I flipped the check to check against `stopped`.